src/backend/distributed/planner/multi_explain.c

```diff
@@ -199,6 +200,17 @@ CitusExplainScan(CustomScanState *node, List *ancestors, struct ExplainState *es
 		return;
 	}
 
+	if (AlterTableInProgress())
```

Comment:
I think this definitely deserves a comment. When this happens? Why cannot we support it?